src/windows/gfx/shaders/d3d11/ps.ps4

```diff
@@ -42,6 +45,39 @@ inline float4 yuv_to_rgba(float y, float u, float v)
 	return float4(r, g, b, 1.0);
 }
 
+inline float4 yuv_to_rgba_rec2020(float y, float u, float v)
+{
+	// Using "RGB to YCbCr color conversion for UHDTV" (ITU-R BT.2020)
+	// Thanks to https://github.com/colour-science/colour/blob/c3735e5d0ad67443022ece0b42b575e040eb61d1/colour/models/rgb/ycbcr.py#L472
+
+	const float Y_min = 0.06256109f;
+	const float Y_max = 0.91886608f;
+	const float Y_minmax_dt = Y_max - Y_min;
+	const float C_min = 0.06256109f;
+	const float C_max = 0.93841642f;
+	const float C_minmax_2 = (C_min + C_max) * 0.5f;
+	const float C_minmax_dt = C_max - C_min;
+	
+	y = y - Y_min;
+	u = u - C_minmax_2;
+	v = v - C_minmax_2;
+
+	y = y * 1.0f / Y_minmax_dt;
+	u = u * 1.0f / C_minmax_dt;
+	v = v * 1.0f / C_minmax_dt;
```

Comment:
Why is there a multiply by 1? In C it's multiplication, then division, is this different in shader languages?